### ISSUE TITLE
DF Module Buttons no longer hides buttons when switching layers (found with DF Curvy Walls)

### ISSUE THREAD

**Issue by kagedansa** [flamewave000/dragonflagon-fvtt#431]
**Module**
DF Module Buttons 1.4.2

**Describe the issue**
There are errors in the console log associated with "DF Curvy Walls 3.4.0".
The visible effect in Foundry (V10.291) is that module buttons are no longer hidden when switching to other layers.

**To Reproduce**
1. Switch to wall layer
2. Switch to other layer

**Screenshots (Optional)**
![grafik](https://user-images.githubusercontent.com/110729103/205925419-005ddf10-4516-44be-bb15-9a9d32f6152f.png)
![grafik](https://user-images.githubusercontent.com/110729103/205925512-0e652276-a247-49f1-be17-bda4232d2636.png)
![grafik](https://user-images.githubusercontent.com/110729103/205925592-6aca0344-23e6-4584-bc45-2a52a5feeda1.png)
![grafik](https://user-images.githubusercontent.com/110729103/205925661-c36b202d-e593-4039-ae0c-91abdcc4b628.png)

**Comment by Cryotechnik** [comment]
Commenting to get updates.

**Comment by NelanaSnow** [comment]
Noticing the exact same error, Even with no other modules installed.

**Comment by tjlanza** [comment]
I suspect this is caused by the enhancement added in 1.4.1; I rolled back to 1.4.0 manually and the buttons hide as expected.

**Comment by DrMcCoy** [comment]
Yeah, commit c8d6169dba29e2bdb54e8f7486fef6a5bc57bf1b is to blame. When I do a git revert on it in the current main, build and copy it, the buttons vanish again like they should.

**Comment by flamewave000** [comment]
Fixed in **Library: DF Module Buttons** v2.0.2 for FoundryVTT v12
